modules/integrate-with-anchore-scanner.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * integration/integrate-with-image-vulnerability-scanners.adoc
+:_module-type: PROCEDURE
+[id="integrate-with-anchore-scanner_{context}"]
+= Integrating with Anchore scanner
+
+You can integrate {product-title} with Anchore scanner.
+
+.Prerequisites
+* You have your Anchore scanner username and password for authentication.
```

Comment:
You could potentially change "your" to "an" for consistency with the other prerequisites in this assembly.